anyway/backend_constants.py

```diff
@@ -52,6 +52,10 @@ class DriverType:
         PRIVATE_VEHICLE_DRIVER = 2
         OTHER_DRIVER = 3
 
+    # This class should be correlated with the Roles2 table
+    class Roles2Names:
+        Admins = "admins"
+
```

Comment:
1.  What functionality this class holds? Will Enum suffice here?
2.  I prefer to work with the Role ID in the code, and change it to the name only for printing or outside API.

Reply:
This class is used to access the hard coded roles in the DB (like the "admins"), Enum will suffice here.

The problem with Role ID is that some of the roles will be create dynamically by the user, so it will be easier for the user to refer to their roles by name(like "ynet"), I can create a translation table - i wrote more about this in the comment about `class Roles2(Base)`, as it the same let's continue the talk about the access way to the DB and API there.